code/index.html

```diff
@@ -0,0 +1,159 @@
+                <h4>sticker 3</h4>
+                <p>$price</p>
+              </a>
+            </div>
+            <div class="one-forth column omega thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>sticker 4</h4>
+                <p>$price</p>
+              </a>
+            </div>
+            <div class="one-forth column alpha thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>Shirt 1</h4>
+              </a>
+            </div>
+            <div class="one-forth column thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>Shirt 2</h4>
+              </a>
+            </div>
+            <div class="one-forth column thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>Shirt 3</h4>
+              </a>
+            </div>
+            <div class="one-forth column omega thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>Shirt 4</h4>
+              </a>
+            </div>
+            <div class="clear"></div>
+          </div> <!--eight_colums-->
+        </div> <!--#products-->
+
+        <div id="follow_us clear">
+          <div class="six_colums">
+            <div class="one-third column alpha thumbnail"><img src="http://placehold.it/200x200">
+              <a href="url" title="">
+                <p>instagram</p>
+              </a>
+            </div>
+            <div class="one-third column thumbnail"><img src="http://placehold.it/200x200">
+              <a href="url" title="">
+                <p>instagram</p>
+              </a>
+            </div>
+            <div class="one-third column omega thumbnail"><img src="http://placehold.it/200x200">
+              <a href="url" title="">
+                <p>instagram</p>
+              </a>
+            </div>
+            <div class="one-third column alpha thumbnail"><img src="http://placehold.it/200x200">
+              <a href="url" title="">
+                <p>instagram</p>
+              </a>
+            </div>
+            <div class="one-third column thumbnail"><img src="http://placehold.it/200x200">
+              <a href="url" title="">
+                <p>instagram</p>
+              </a>
+            </div>
+            <div class="one-third column omega thumbnail"><img src="http://placehold.it/200x200">
+              <a href="url" title="">
+                <p>instagram</p>
+              </a>
+            </div>
+            <div class="clear"></div>
+          </div> <!--six_colums-->
+        </div> <!--#follow_us-->
+        <div id="contact">
+        </div> <!--#contact-->
+      </div> <!-- container -->
+    </section> <!-- main -->
+
+    <!------------ FOOTER ------------->
+    <footer>
+
+      <div class="logo">Sweaty Palm Stickers</div>
```

Comment:
This is nearly impossible to read on the current background - try switching it up!